public/apps/configuration/panels/role-list.tsx

```diff
@@ -137,6 +139,22 @@ export function RoleList(props: AppDependencies) {
     fetchData();
   }, []);
 
+  const handleDelete = async () => {
+    const rolesToDelete: string[] = selection.map(r => r.role_name);
+    try {
+      await requestDeleteRoles(props.coreStart.http, rolesToDelete);
+      // Refresh from server (calling fetchData) does not work here, the server still return the roles
+      // that had been just deleted, probably because ES takes some time to sync to all nodes.
+      // So here remove the selected roles from local memory directly.
+      setRoleData(difference(roleData, selection));
+      setSelection([]);     
+    } catch (e) {
+      console.log(e);
```

Comment:
should we promote user to proper action to do in case of exception

Reply:
Considering pop up a toast to let user know the failure. Not included in this PR.